io-engine/src/lvs/lvol_snapshot.rs

```diff
@@ -550,7 +559,20 @@ impl SnapshotOps for Lvol {
     async fn snapshot_iter(self) -> LvolSnapshotIter {
         LvolSnapshotIter::new(self)
     }
-
+    /// Destroy snapshot.
+    async fn destroy_snapshot(mut self) -> Result<(), Self::Error> {
+        if self.list_clones_by_snapshot_uuid().is_empty() {
```

Comment:
Here we can later add blob inflation for cases when there is exactly one clone exists.

Reply:
okay, I will explore this option to use of `spdk_bs_blob_decouple_parent` function to decouple snapshot from clone and blob inflation.
https://dcsw-bolt.atlassian.net/browse/GTM-1148